src/EditorFeatures/VisualBasic/QuickInfo/SemanticQuickInfoProvider.vb

```diff
@@ -143,7 +148,9 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.QuickInfo
                 Return Me.CreateClassifiableDeferredContent(contentBuilder)
             End If
 
-            Return Await CreateContentAsync(document.Project.Solution.Workspace, token, semantics, types, supportedPlatforms:=Nothing, cancellationToken:=cancellationToken).ConfigureAwait(False)
+            Return Await CreateContentAsync(
```

Comment:
why are you passing Empty for the capture nodes here?  Needs comment explaining why.